src/index.html

```diff
@@ -7,6 +7,435 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="moyo_logo"
+            >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_item">
+            <a href="#apple" class="header__nav_link is-active">Apple</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#samsung" class="header__nav_link">Samsung</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#smartphones" class="header__nav_link">Smartphones</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#laptops&computers"
+              class="header__nav_link"
+              data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#gadgets" class="header__nav_link">Gadgets</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#tablets" class="header__nav_link">Tablets</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#photo" class="header__nav_link">Photo</a>
+          </li>
+
+          <li class="header__nav_item">
+            <a href="#video" class="header__nav_link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div class="card" data-qa="card">
+          <img
+          class="card_img"
```

Comment:
Change class name from 'card_img' to 'card__img' to follow BEM naming convention.